src/index.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+// import "babel-polyfill";
```

Comment:
Well cra isn't broken in IE, it just doesn't bundle the polyfills our code needs. The first one it breaks on is `Object.values` but I'm guessing there are quite a few other ones we are using.